docs/freebsd/README.md

```diff
@@ -61,12 +61,36 @@ autoboot_delay="-1"
 console="comconsole"
 __EOF__
 ```
-Fetch a copy of the FreeBSD kernel sources and place them in `/usr/src`.  For instance, to get a copy of the current development sources, run:
-
+Fetch a copy of the FreeBSD kernel sources and place them in `/usr/src`. You will likely need to expand the disk of the VM. Make sure the VM is off. On the host:
+```console
+# truncate -s 15G $IMAGEFILE
+```
+After booting VM back up, run:
+```console
+# /etc/rc.d/growfs onestart
```

Comment:
I think the instructions on growing the VM image should come first: if you expand the disk image immediately after downloading, i.e., the VM was never booted, /etc/rc.d/growfs should handle it automatically. It runs only on first boot.